### ISSUE TITLE
IBM AfroHacks at AfroTech Participation

### ISSUE THREAD

**Issue by jritten** [pytorch/pytorch#112318]
### 🚀 The feature, motivation and pitch

IBM AfroHacks at AfroTech Hackathon starts on Monday, and we would love to drive contributions to the PyTorch project repo(s), but they are not properly labeled for participants wanting to contribute to Ray during the 2-day hackathon which projects/issues to work on. 

Add “afrohacks” to Settings => Topics on any Carbon repos, and add "afrohacks" to issue Labels, to indicate the repos/repo issues Ray would like to have participants work on during the 48-hour AfroHacks at AfroTech Hackathon starting on Monday. Please feel free to connect with me with any questions. Thank you!

**Comment by jritten** [comment]
https://ibm.biz/afrohacks-hackathon

**Comment by malfet** [comment]
@jritten Thank you for the issue, but wouldn't it be better to announce something like that in https://dev-discuss.pytorch.org/

**Comment by jritten** [comment]
@malfet I'm not sure where the best place to announce this would be as I'm new to the PyTorch community. the event starts tomorrow with hacking starting tomorrow evening. I would appreciate any help with this. is PyTorch looking for contributions? we'll have 200+ participants at our event contributing to a handful of open source projects. thank you!

**Comment by malfet** [comment]
@jritten yes, please feel free to post to dev-descuss as well as to announcements channel on Slack. And contributions are always welcome.
